lighthouse-core/report/html/report-styles.css

```diff
@@ -645,7 +666,10 @@ details, summary {
 .lh-clump > .lh-audit-group__description,
 .lh-audit-group--diagnostics .lh-audit-group__description,
 .lh-audit-group--opportunities .lh-audit-group__description,
-.lh-audit-group--metrics .lh-audit-group__description {
+.lh-audit-group--metrics .lh-audit-group__description,
+.lh-audit-group--pwa-fast-reliable .lh-audit-group__description,
```

Comment:
this is getting dicey, maybe we should think about an overall `--decorated` class or something

Reply:
> this is getting dicey

yeah, agreed. Ideally we can get clumps off of group classes, then we'd just need to update the perf renderer